docs/dashboard.md

```diff
@@ -0,0 +1,33 @@
+# SC4S Metrics and Events Dashboard
+SC4S Metrics and Events Dashboard enables monitoring crucial metrics and events flow of all the SC4S instances sending data to a chosen Splunk platform.
+
+
+## Functionalities
+
+### Overview metrics
+The dashboard displays the cumulative sum of received and dropped messages of all SC4S instances in chosen interval and for the specified time range. It is 30 seconds span in a 15 minutes window by default.
+
+Received messages panel can be used as a heart beat metric, since a healthy SC4S instance should send at least one message per 30 seconds. This is a metrics message and it's included in the count.
+
+Dropped messages panel should remain at a constant level of 0. If SC4S drops messages due to filters, slow performance or other reasons, the number of dropped messages will be there until the instance restart. This does not include potential UDP messages dropped from the port buffer, which SC4S is completely unaware of.
+
+### Single instance metrics
+The user can display instance name and SC4S version for a chosen SC4S instance.
+SC4S version is available in versions >=3.15.0.
+
+The dashboard also displays a timechart of deltas of received, queued and dropped messages for a chosen SC4S instance.
```

Comment:
The dashboard also displays a timechart of deltas for received, queued, and dropped messages for a chosen SC4S instance.